ponylang-mode.el

```diff
@@ -104,7 +104,7 @@
 
 (defconst ponylang-keywords
   '("actor"
-    "be" "break"
+    "as" "be" "break"
```

Comment:
'ive been keeping this in alphabetical order, one line per letter.

can you move "as" up to the same line as "actor" and have it appear after it?